### PR TITLE
add description and name.

### PR DESCRIPTION
**Pull request by SupersnipsXDC**
please upload our logo from link: https://ibb.co/YczxM8T

### CODE REVIEW

File: chains/razorSChain/validatorList.json.bak [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+   {
+    "validators": {
```
Comment (SkandaBhat) [inline chains/razorSChain/validatorList.json.bak]: Please remove this .bak file

File: chains/razorSChain/validatorList.json [hunk 2]
```diff
@@ -36,6 +36,12 @@
             "name": "FutureWave",
             "website": "https://futurewave.sh/",
             "logo": "futurewave.jpg"
+        },
+        "20": {
+            "address": "0x17404D7880d305b3E65698b5551fB21639294CeB",
+            "name": "CommunityStake",
+            "website": "coming soon",
+            "logo": "CommunityStake.jpg"
```
Comment (SkandaBhat) [inline chains/razorSChain/validatorList.json]: Please add `CommunityStake.jpg` image in the assets folder.

**Review by SkandaBhat — DISMISSED** [review]

File: chains/razorSChain/validatorList.json.bak (hunk 1 above)
Comment (SupersnipsXDC) [reply chains/razorSChain/validatorList.json.bak]: done

File: chains/razorSChain/validatorList.json (hunk 2 above)
Comment (SupersnipsXDC) [reply chains/razorSChain/validatorList.json:44]: Uploads are disabled. getting error- 
"
81541_BCard_TU_BACK (1)
File uploads require push access to this repository. "

**Review by SupersnipsXDC — COMMENTED** [review]
edited-

**Review by SupersnipsXDC — COMMENTED** [review]
done

File: chains/razorSChain/validatorList.json (hunk 2 above)
Comment (SupersnipsXDC) [reply chains/razorSChain/validatorList.json]: done

**Review by SupersnipsXDC — COMMENTED** [review]
.

**Comment by hrishikeshio** [comment]
Remove website or add valid website.

**Review by SupersnipsXDC — COMMENTED** [review]
Done

**Review by hrishikeshio — DISMISSED** [review]

**Review by hrishikeshio — DISMISSED** [review]

**Comment by hrishikeshio** [comment]
@SupersnipsXDC All commits must be signed, please check https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

**Comment by SupersnipsXDC** [comment]
added communitystake.jpg which is verified now

**Comment by SupersnipsXDC** [comment]
closed, made fresh and clean pr.

**Pull request closed without merging**